### PR TITLE
Outlaw Adjustments: QoL, anti grief, etc

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request
Ignore the shoddy commit stuff, I hate gitkraken.
Anyway, this commit prevents easier ways to grief round start while also giving outlaws goodies and other stuff that are quality of life changes i.e headsets, etc

## Why It's Good For The Game

Round start grenades bad.

## Pre-Merge Checklist

- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog

:cl:
tweak: tweaked a few things
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -263,10 +264,8 @@ Raider
 	head = /obj/item/clothing/head/helmet/f13/raider/blastmaster
 	backpack_contents = list(
 		/obj/item/kitchen/knife/butcher = 1,
-		/obj/item/grenade/homemade/firebomb = 4,
-		/obj/item/bottlecap_mine = 1,
```
[the remark this answers is not in the document]
Comment (AllCorruptingWyrm) [reply code/modules/jobs/job_types/wasteland.dm:267]: Certain people are explicitly using blastmaster to grief, that's how they've been doing this. Bombs shouldn't be something they get round start.

[the remark this answers is not in the document]
Comment (AllCorruptingWyrm) [reply code/modules/jobs/job_types/wasteland.dm:267]: re: bombs
If they want them, staff are able to grant items for gimmicks.

File: code/modules/jobs/job_types/wasteland.dm [hunk 2]
```diff
@@ -374,6 +380,8 @@ Raider
 		/obj/item/gun/ballistic/revolver/m29/snub = 2,
 		/obj/item/storage/belt/holster = 1,
 		/obj/item/ammo_box/m44 = 3,
+		/obj/item/radio/headset = 1,
+		/obj/item/radio/headset = 1,
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/wasteland.dm:384]: Two radio headsets here, don't know if it's intended or a bug, just saying.

File: code/modules/jobs/job_types/wasteland.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/wasteland.dm:267]: Why not? They can get 2 bombs and cause no damage, perhaps 9 bombs were too much but cutting them from 9 to 0 is extreme for a couple of griefers.

File: code/modules/jobs/job_types/wasteland.dm (hunk 2 above)
Comment (AllCorruptingWyrm) [reply code/modules/jobs/job_types/wasteland.dm]: fixed, if you see anything else don't hesitate to say.

File: code/modules/jobs/job_types/wasteland.dm [hunk 3]
```diff
@@ -289,13 +290,12 @@ Raider
 	suit = /obj/item/clothing/suit/armor/f13/raider/badlands
 	head = /obj/item/clothing/head/helmet/f13/fiend
 	backpack_contents = list(
-		/obj/item/gun/energy/laser/wattz/magneto = 1,
-		/obj/item/gun/energy/laser/wattz = 1,
+		/obj/item/gun/energy/laser/wattz/magneto = 2,
```
Comment (panzerr1944) [inline code/modules/jobs/job_types/wasteland.dm:293]: you cant dual-wield magnetos.

Comment (AllCorruptingWyrm) [reply code/modules/jobs/job_types/wasteland.dm:293]: raiders as a whole did lose some of their goodies, people should want to be regular bumfuck fiends and have a lore fitting loadout for it, not uber armored commandos like every round.

**Pull request merged**